p2p/address_utils.go

```diff
@@ -44,3 +44,14 @@ func stringsToPeerInfos(peers []string) ([]peer.AddrInfo, error) {
 	}
 	return pinfos, nil
 }
+
+func convIdToStringArray(peers []peer.ID) []string {
```

Comment:
Please change to `PeerIdToStringArray`. The fact it is not exported make me wonder if you're still using it?